### ISSUE TITLE
Warning for VaInput 'Failed to resolve component: va-icon'

### ISSUE THREAD

**Issue by DallasP9124** [epicmaxco/vuestic-ui#1486]
**Vuestic-ui version:** 1.3.4

**Steps to reproduce**
Create a basic project and use the VaInput component somewhere.

**What is the expected behavior?**
That this warning does not show when using the component.

**What is the current behavior?**
The warning appears.

**Other information**
This is the exact warning
```
[Vue warn]: Failed to resolve component: va-icon
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <VaInput disabled=false readonly=false modelValue=""  ... > 
```
This is the usage:
```vue
<VaInput
    placeholder="Filter..."
    v-model="tableFilter"/>
```

**Comment by syahrizalxs** [comment]
Can I take this issue?

**Comment by RVitaly1978** [comment]
This issue has been resolved in [PR#1383](https://github.com/epicmaxco/vuestic-ui/pull/1383)
